app/(candidate)/dashboard/door-knocking/components/DoorKnockingPage.js

```diff
@@ -1,25 +1,66 @@
-import { serverFetch } from 'gpApi/serverFetch';
+'use client';
 import DashboardLayout from '../../shared/DashboardLayout';
 import InteractionsSummary from './InteractionsSummary';
 import { apiRoutes } from 'gpApi/routes';
 import InteractionsSummaryPie from './InteractionsSummaryPie';
 import InteractionsByDay from './InteractionsByDay';
 import RatingSummary from './RatingSummary';
+import H1 from '@shared/typography/H1';
+import { dateWithTime } from 'helpers/dateHelper';
+import Button from '@shared/buttons/Button';
+import { syncEcanvasser } from 'app/admin/ecanvasser/components/EcanvasserCard';
+import { clientFetch } from 'gpApi/clientFetch';
+import { useEffect, useState } from 'react';
+import Body2 from '@shared/typography/Body2';
 
 async function fetchEcanvasserSummary() {
-  const response = await serverFetch(apiRoutes.ecanvasser.mySummary);
+  const response = await clientFetch(apiRoutes.ecanvasser.mySummary);
   return response.data;
 }
 
-export default async function DoorKnockingPage(props) {
-  const summary = await fetchEcanvasserSummary();
+export default function DoorKnockingPage(props) {
+  const [summary, setSummary] = useState(null);
+  const [isSynching, setIsSynching] = useState(false);
+  useEffect(() => {
+    fetchSummary();
+  }, []);
+
+  const fetchSummary = async () => {
+    const data = await fetchEcanvasserSummary();
+    setSummary(data);
+  };
+
```

Comment:
Why go through all this effort to make this initial hydration client-side?

Why not just fetch the initial data in `door-knocking/page.js`, and pass that in as an initial value in props to hydrate the `useState` with? 🤔